ci/test_cpp_common.sh

```diff
@@ -18,6 +18,9 @@ set +u
 conda activate test
 set -u
 
+RAPIDS_CUDA_MAJOR="${RAPIDS_CUDA_VERSION%%.*}"
+LIBRMM_CHANNEL=$(rapids-get-artifact ci/rmm/pull-request/1223/042a67e/rmm_conda_cpp_cuda${RAPIDS_CUDA_MAJOR}_$(arch).tar.gz)
```

Comment:
Maybe this also needs an update?

```suggestion
LIBRMM_CHANNEL=$(rapids-get-artifact ci/rmm/pull-request/1223/1901c8c/rmm_conda_cpp_cuda${RAPIDS_CUDA_MAJOR}_$(arch).tar.gz)
```